.github/workflows/deepx-ci.yml

```diff
@@ -0,0 +1,28 @@
+name: DeepX CI
+
+run-name: Build Docker container and perform unit testing
+
+on: [push, workflow_dispatch]
```

Comment:
Do we want to run it on every `push` in any branch? Should we also include `pull_request`?

Reply:
Good point. `push` should be on the main branch only, which is `ign-rendering6` for now. Also should be activated on PR.

Changed it as follows.
```
on: 
  push:
    branches: [ign-rendering6]
  pull-request:
    branches: [ign-rendering6]
  workflow_dispatch:
```